src/monodroid/jni/jni-wrappers.hh

```diff
@@ -123,7 +130,8 @@ namespace xamarin::android
 		explicit jstring_array_wrapper (JNIEnv *env) noexcept
```

Comment:
@grendello: should we just *remove* this constructor?  I don't see it used anywhere, and if it *had* been used, we'd have had a runtime crash because `wrappers` would have been garbage.

Reply:
My PR [makes uses of it](https://github.com/xamarin/xamarin-android/pull/3734/files#diff-09282c2637d665875eb937b5222b94f7R2077) now to essentially have a `nullptr` array